sources/dialect/README.MD

```diff
@@ -0,0 +1,19 @@
+# Dialect Support
+
+This rule has net yet been added.
+For now, we have only added some files and sources.
+This section will gradually be expanded as the rule is written.
+
+The files inside ``original_files`` were added from the version 1 tree of the original repository, [``en-wl/wordlist``](https://github.com/en-wl/wordlist/tree/v1).
```

Comment:
I'm surprised by the double back tick. Shouldn't it be this?


```suggestion
The files inside `original_files` were added from the version 1 tree of the original repository, [`en-wl/wordlist`](https://github.com/en-wl/wordlist/tree/v1).
```
I would expect double backtick only when using a backtick in the quoted text

More information here

https://stackoverflow.com/questions/55586867/how-to-put-in-markdown-an-inline-code-block-that-only-contains-a-backtick-char